src/blocks/packages/pacman.rs

```diff
@@ -0,0 +1,276 @@
+//! Pending updates available on pacman or an AUR helper.
```

Comment:
See previous note about docs.

Reply:
I can clear the other stuff like configuration, example and add them in `packages.rs` file but some of the info like pacman has like hooking and some config setup for apt which user need to know and that is very specific so ig can't put it in package.rs file

Reply:
So if you could tell where i can put that and clear the documentation of submodule

Reply:
I'd use https://github.com/greshake/i3status-rust/blob/master/src/blocks/weather.rs and https://github.com/bim9262/i3status-rust/blob/privacy/src/blocks/privacy.rs as exmples where service/driver specific configurations are described in the block doc.

Reply:
Done